### ISSUE TITLE
No customers meta?

### ISSUE THREAD

**Issue by gianlnicc** [judgej/wc-api-custom-meta#19]
I need to retrieve meta data associated to customers (vat a second email), but I see no results for this.
This only works for products meta?

**Comment by judgej** [comment]
Yes, it is only for products, designed to fill a gap (my external fulfilment system needed access to additional metdata for the products to help decide who was going to fulfil it. As such, the standard customer data was enough for that process.

**Comment by judgej** [comment]
Use the new v2 REST API (WC 3.x) instead of this plugin. It now does everything this plugin does out of the box. The README gives some examples of how you would manage the metadata fields.
